wostools/cli.py

```diff
@@ -1,26 +1,69 @@
 import click
-
-import networkx
-
 from wostools import CollectionLazy
+from wostools.fields import FIELDS
+import json
 
 
 @click.group()
 def main():
     """
     A little cli for wos tools.
     """
-    click.echo("Never fear wostools cli is here")
 
 
-@main.command("citation-graph")
-@click.argument("source", type=click.File("r"), nargs=-1)
-@click.argument("output", type=click.File("w"), nargs=1)
-def citation_graph(source, output):
+@main.command("citation-pairs")
+@click.argument("sources", type=click.File("r"), nargs=-1)
+@click.option("--output", type=click.File("w"), show_default=True, default=None)
```

Comment:
This can be `default="-"`, then `output.write` would write to the standard output similar to `print`.